jib-core/src/main/java/com/google/cloud/tools/jib/builder/EntrypointBuilder.java

```diff
@@ -37,6 +37,7 @@
     ImmutableList<String> classPaths =
         ImmutableList.of(
             sourceFilesConfiguration.getDependenciesPathOnImage() + "*",
+            sourceFilesConfiguration.getSnapshotDependenciesPathOnImage() + "*",
```

Comment:
Is there any harm to putting the snapshot dependencies in the same `/app/libs/` directory?

Reply:
We also might not want to have the snapshot layer if there are no snapshot dependencies, so having that directory added to the classpath may cause directory not found exceptions?

Reply:
Yeah, I don't have any reason reason for picking a separate directory. Let me change it.